packages/react/src/primitives/types/checkbox.ts

```diff
@@ -6,7 +6,14 @@ export interface CheckboxProps extends FlexProps, InputProps {
   /**
    * The label text
    */
-  children: FieldProps['label'];
+  label: FieldProps['label'];
+
+  /**
+   * Visually hide label (not recommended in most cases)
+   * @default false
+   */
+  labelHidden?: boolean;
```

Comment:
@reesscot @hvergara should this be isLabelHidden? It looks like field, and search field use `labelHidden`, switchField uses `isLabelHidden`

Reply:
Yeah, I think there's an issue in the backlog to update them all to `isLabelHidden`